skaffold.yaml

```diff
@@ -1183,8 +1185,8 @@ profiles:
   - command: debug
   patches:
   - op: add
-    path: /deploy/helm/releases/0/setValues
+    path: /deploy/helm/releases/0/setValueTemplates
     value:
       replicaCount: 1
       config.leaderElection.leaderElect: false
-      config.server.healthProbes.enable: false
+      config.server.healthProbes.enable: "{{ default \"\" .SKAFFOLD_DISABLE_HEALTH_CHECKS | lower | splitList \",\" | has \"gardenlet\" | ternary \"false\" \"true\" }}"
```

Comment:
Please see my first comment about `SKAFFOLD_DISABLE_HEALTH_CHECKS` env.

Reply:
May we merge this into the discussion above?

While this template looks complex, all it's doing is creating a truthy value on whether the string contains the particular component name. This would be a trivial reference if there was a different environment variable for each component, but the user experience complexity of this is reduced if the developer can look at a single environment variable to see what is enabled instead of several. I am working with the Skaffold team on global templates within a configuration, all the new template instantiations would become very short with that in place.